src/views/facetnavigation.tsx

```diff
@@ -85,7 +85,6 @@ export const FacetNavigationFrame: Component<{ children: JSX.Element }> = (
                     "text-card-foreground": true,
                     "shadow-sm": true,
                     fixed: true,
-                    "z-40": true,
```

Comment:
Thanks! I think at one point this was needed to make the nav menu in mobile view appear above the main content, but that doesn't seem necessary anymore.